amxmodx/CPlugin.h

```diff
@@ -86,6 +90,8 @@ class CPluginMngr
 		inline void setTitle(const char* n) { title = n; }
 		inline void setAuthor(const char* n) { author =n; }
 		inline void setVersion(const char* n) { version = n; }
+		inline void setUrl(const char* n) { url = n; }
```

Comment:
BTW. AMX tells you string length, so you potentially exploit it for faster owned string construction. But I think it isn't very important

Reply:
@In-line I did everything the way it was done before me (title, version, author), that not to refactor someone else's code (I have no desire to do this)